public/app/features/explore/AddToDashboard/AddToDashboardModal.test.tsx

```diff
@@ -95,7 +95,9 @@ describe('Add to Dashboard Modal', () => {
 
       userEvent.click(screen.getByRole('button', { name: /save and keep exploring/i }));
 
-      expect(await screen.findByRole('alert')).toHaveTextContent('name exists');
+      expect(await screen.findByRole('alert')).toHaveTextContent(
+        'A dashboard with the same name already exists in this folder.'
```

Comment:
@grafana/observability-experience-squad i'm thinking that maybe we shouldn't test for text content at all (aside maybe for when we are mocking BE responses and we know we'll be using messages returned by the BE) as they are subject to change and in the future, with i18n in place they may not even make any sense. wdyt?

Reply:
In the past (not saying this is right or wrong), translations would work by having key/values - so like this might be in a file for English might be like `explore.errors.dashboard.nameExists: A dashboard with the same name already exists in this folder.` 

For testing, we would mock the call to the translation layer that pulls the right string for the key, and detect the right key was passed in to the mock. When we eventually move our strings into translation files like this, I could see changing the tests being a part of that work. This also means if we change wording on these things after that point but keep the same key, the test won't be affected.

I am not strongly opinionated on this but I would leave it to test that we are getting the right error for the test and we can remove it if it becomes too cumbersome to keep or change it when we add translations.

Reply:
good point, i think that's a reasonable approach, i'll leave them there 👍